utils/dataloaders.py

```diff
@@ -0,0 +1,84 @@
+import random
+
+import torch
+import numpy as np
+from omegaconf import DictConfig
+from datasets import load_dataset
+from torch.utils.data import DataLoader
+
+
+def normalize_batch(batch):
```

Comment:
There's already `BatchNorm` in both models, does this add anything?

Reply:
That's what I've realised a few minutes earlier. Tried to apply https://github.com/soumith/ganhacks to our case, probably used a wrong kind of normalization as well. I was thinking that normalizing the input in -1, 1 range could help the generator, since we're using tanh.

Reply:
it didn't really and broke the discriminator as an addition [oops](https://wandb.ai/sjanas/ECG%20GAN/runs/y920whzc?workspace=user-sneakyinsect)